CHANGELOG.md

```diff
@@ -6,6 +6,62 @@ For advice on how to use these release notes see [our guidance on staying up to
 
 ### New features
 
+#### Update to the new GOV.UK logo
+
+The GOV.UK logo has been updated to reflect the changing of the monarch; King Charles III uses the Tudor Crown, rather than the St Edward’s Crown chosen by Queen Elizabeth II.
+
+If your service uses GOV.UK branding, you should update your service to use the new logo as soon as possible.
+
+These changes were made in the following pull requests:
+
+- [#4354: Implement the Tudor crown favicons (v5.x)](https://github.com/alphagov/govuk-frontend/pull/4354)
+- [#4297: Implement the Tudor crown in the Header component (v5.x)](https://github.com/alphagov/govuk-frontend/pull/4297)
+
+##### Include the new logo assets
+
+Multiple new image assets are included in this release. These will need to be copied to your service's image assets folder if they aren't being used directly from the Frontend package. By default this folder is located at `/assets/images`.
+
+If you are using Nunjucks, the asset path may have been changed by the `assetPath` global variable or `assetsPath` parameter on the header component.
+
+Copy the following files from `/dist/assets/images` into your assets folder. Any images with the same name as an existing image can be safely overwritten.
+
+- favicon.ico
+- favicon.svg
+- govuk-icon-180.png
+- govuk-icon-192.png
+- govuk-icon-512.png
+- govuk-icon-mask.svg
+- govuk-opengraph-image.png
+
+##### Update the logo in the header of your page
+
+If you are using the `govukHeader` Nunjucks macro in your service, add the `useTudorCrown` parameter to the macro instantiation.
```

Comment:
Shall we note the future default change here?

```suggestion
If you are using the `govukHeader` Nunjucks macro in your service, add the `useTudorCrown` parameter to the macro instantiation. This will become the default in a future version of GOV.UK Frontend.
```